landoapi/commit_message.py

```diff
@@ -109,25 +89,6 @@
 )
 RE_SOURCE_REPO = re.compile(r"^Source-Repo: (https?:\/\/.*)$", re.MULTILINE)
 RE_SOURCE_REVISION = re.compile(r"^Source-Revision: (.*)$", re.MULTILINE)
-# Like BUG_RE except it doesn't flag sequences of numbers, only positive
-# "bug" syntax like "bug X" or "b=".
-BUG_CONSERVATIVE_RE = re.compile(r"""(\b(?:bug|b=)\b(?:\s*)(\d+)(?=\b))""", re.I | re.X)
```

Comment:
This string is slightly different than the one on line 35, and this is the line that was prior to this change would have overridden the original value. Is there any functional difference between the two? Just wanted to flag this in case there is.

Reply:
Looks like this version has an extra fix from your patch in [D69608](https://phabricator.services.mozilla.com/D69608). I will switch this to removing the other copy of `BUG_CONSERVATIVE_RE`. :)